src/oidcop/session/claims.py

```diff
@@ -95,6 +95,10 @@ def get_claims(self, session_id: str, scopes: str, claims_release_point: str) ->
         claims.update(base_claims)
 
         # Scopes can in some cases equate to set of claims, is that used here ?
+        if not _context.cdb[client_id].get("add_claims_by_scope") is None:
+            module.kwargs["add_claims_by_scope"] = \
+                _context.cdb[client_id].get("add_claims_by_scope")
+
         if module.kwargs.get("add_claims_by_scope"):
```

Comment:
You shouldn't overwrite the modules kwargs, this will affect later calls.

I think this would be cleaner:
```python
if "add_claims_by_scope" in _context.cdb[client_id]:
    add_claims_by_scope = _context.cdb[client_id]["add_claims_by_scope"]
else:
    add_claims_by_scope = module.kwargs.get("add_claims_by_scope"):

if add_claims_by_scope:
    ...
```

Reply:
yes "if not is None" is heavy

Reply:
Ok, done, great!